src/legacyCompat.ts

```diff
@@ -60,7 +60,13 @@ export async function legacyGetMapFromParams(
           throw new Error('Processing API is only possible with SH V3 layers');
         }
         if (evalscript) {
-          const decodedEvalscript = atob(evalscript);
+          let decodedEvalscript;
+          if (typeof window !== 'undefined' && window.atob) {
+            decodedEvalscript = atob(evalscript);
+          } else {
+            // node.js doesn't support atob:
+            decodedEvalscript = Buffer.from(evalscript, 'base64').toString('ascii');
```

Comment:
Should it be `utf-8` instead of `ascii`?

Reply:
Agreed, probably best to use UTF-8 and be explicit about it (I have now added encoding to "`btoa`" too).

Reply:
Ok, resolving.